### DISCUSSION TITLE
dragging and dropping skin files into skin editor doesn't work

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#24839] [Q&A] [answered]
there (apparently) existed a feature where you could drag and drop skin files into the skin editor and it'd update the skin accordingly. (source https://github.com/ppy/osu/discussions/24241#discussioncomment-6461865 and https://twitter.com/ppy/status/1509845390908325894), but it simply doesn't seem to work

i [asked on discord](https://discord.com/channels/188630481301012481/188630652340404224/1153062263159259259) about this and was told it's a regression

https://github.com/ppy/osu/assets/36758269/e01df9ea-31f5-4fb5-ba78-d5f226503dad

when dragging the hit circle or cursor into the game, the expectation is of course for all hit circles and the cursor to update properly. at the moment it sometimes also doesn't display the image that was dragged in, but instead some other image

**Comment by peppy** [comment, marked as answer]
You didn't drag the `@2x` version. Lazer doesn't use non-`@2x` files when an `@2x` one exists. Make sure to reload the skin, too:

https://github.com/ppy/osu/assets/191335/7a39047c-d21b-4b42-a82f-e4cba6ed2e4d
